src/SFA.DAS.LoginService.Application/Services/UserAccountService.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Threading.Tasks;
+using Microsoft.EntityFrameworkCore;
+using SFA.DAS.LoginService.Application.Interfaces;
+using SFA.DAS.LoginService.Data;
+using SFA.DAS.LoginService.Data.Entities;
+
+namespace SFA.DAS.LoginService.Application.Services
+{
+    public class UserAccountService : IUserAccountService
```

Comment:
Maybe we can improve this name?  It doesn't speak to me why it is different to `UserService`.  

I'm afraid it's not clear to me why we're going directly to the database and bypassing the `UserManager`.

Reply:
I agree, just not sure what to name it

Reply:
We could call it 'WithoutHttpContextUserService' I know it sounds like a joke but it's purpose becomes clear

Reply:
Ok, I see the reason now.

Hmm, how about switching it round and calling the class without HttpContext `UserManager` and the original one `HttpContextUserManager` or something like that?